lib/processors/jsdoc/lib/ui5/plugin.js

```diff
@@ -249,13 +253,19 @@ function analyzeModuleDefinition(node) {
 		currentModule.dependencies = convertValue(args[arg], "string[]");
 		arg++;
 	}
-	if ( arg < args.length && args[arg].type === Syntax.FunctionExpression ) {
+	if ( arg < args.length && 
+		[Syntax.FunctionExpression, Syntax.ArrowFunctionExpression].includes(args[arg].type)) {
+
 		currentModule.factory = args[arg];
 		arg++;
 	}
 	if ( currentModule.dependencies && currentModule.factory ) {
 		for ( let i = 0; i < currentModule.dependencies.length && i < currentModule.factory.params.length; i++ ) {
-			const name = currentModule.factory.params[i].name;
+			const name =
+				(currentModule.factory.params[i].type === Syntax.ObjectPattern)
+					? currentModule.factory.params[i].properties[0].value.name // ObjectPattern means destructuring of the parameter
```

Comment:
What happens if one is destructuring more than one property?
If I understand this code correctly, it will only pick the first one, right?
```js
function({ Foo, Bar }) {}
```